akka-docs/src/main/paradox/stream/operators/index.md

```diff
@@ -502,12 +506,10 @@ For more background see the @ref[Error Handling in Streams](../stream-error.md)
 * [mergeLatest](Source-or-Flow/mergeLatest.md)
 * [mergePreferred](Source-or-Flow/mergePreferred.md)
 * [mergePrioritized](Source-or-Flow/mergePrioritized.md)
-* [mergePrioritizedN](Source/mergePrioritizedN.md)
```

Comment:
why did these disappear?

Reply:
Nice catch! I've no idea how they got removed given that file is auto-generated :confused:.

In any case an `sbt akka-docs/paradox` seems to have fixed it.

However I noticed that also `[preMaterialize](Source-or-Flow/preMaterialize.md)` got added. Probably the index wasn't regenerated when `preMaterialize` has been added to `Flow`?

Reply:
apparently! It'd be nice if we could catch this as part of CI, but anyway it's good that it's added now :)